web/src/gql/provider.tsx

```diff
@@ -42,11 +43,15 @@ const Provider: React.FC<Props> = ({ children }) => {
     }
   });
 
+  const httpLink = new HttpLink({
+    uri: endpoint,
+  });
```

Comment:
I don't think this is necessary. The Apollo docs [here](https://www.apollographql.com/docs/react/api/link/apollo-link-http/) say Apollo uses Httplink by default if we pass the uri option to ApolloClient, which we do.

Reply:
When using `link: ApolloLink.from` option
It does not work, unless I explicitly provide the HttpLink, for now, I'll leave it, and maybe in the future I'll inspect why this happening and check graphQL repo for similar issues